CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to this extension will be documented in this file.
   - "Flink Database Relations" (now the default mode) displays tables, views, and their columns
   - "Flink AI" displays connections, tools, models, and agents
 - Show view definition text in a view's tooltip.
+- "Join Confluent Community Slack" link in the Help Center view to connect with the Confluent community on Slack.
```

Comment:
Minor adjustment for accuracy, I'd suggest either removing the reference or changing it to "Open Confluent Community Slack" to match the tree item:
```suggestion
- Link in the Help Center view to connect with the Confluent community on Slack.
```